bin/run_composable-kernels.sh

```diff
@@ -16,14 +16,15 @@ thisdir=$(dirname $realpath)
 export PATH=$AOMP/bin:$PATH
 
 function printHelp {
+  set +x
```

Comment:
Runaway debug?
I guess this should either be outside of this function or not here at all?